common/Generieke-Datatypen-Gemeenten.yaml

```diff
@@ -0,0 +1,79 @@
+openapi: 3.0.0
+info:
+  title: Generieke datatypen voor Gemeentelijke APIs onder regie van VNGRealistaie
+  description: |
+    De definitie van generieke datatypen die worden gebruikt door gemeentelijke API's.
+    Deze datatypen komen overeen met het basismodel Generieke Datatype Gemeenten waarnaar verwezen wordt vanuit informatiemodellen.
+    Er worden uit dit bestand met herbruikbare componenten geen componenten verwijderd.
+    Als een component in de toekomst niet meer hergebruik mag worden dan wordt het deprecated verklaard.
+  version: '1.0.0'
+  contact:
+    name: Generieke Datatypen Gemeenten.
+    url: https://github.com/VNG-Realisatie/API-Kennisbank/Common
+  license:
+    name: European Union Public License, version 1.2 (EUPL-1.2)
+    url: https://eupl.eu/1.2/nl/
+paths: {}
+components:
+  schemas:
+    UUID:
+      type: "string"
+      minLength: 36
+      maxLength: 36
+      example: "123e4567-e89b-12d3-a456-426614174000"
+    OIN:
+      type: "string"
+      minLength: 20
+      maxLength: 20
+      pattern: ^[0-9]{20}$
+      description: |
+                    Het Organisatie-identificatienummer (OIN) is een uniek nummer dat Logius kan toekennen aan organisaties om zich te kunnen identificeren, authentiseren en of autoriseren bij digitaal berichtenverkeer binnen en met de overheid.
+    DatumOnvolledig:
+      type: "object"
+      description: |
+                    Gegevens over de datums die mogelijk niet volledig zijn.
```

Comment:
Maak de description in model en in dit schema gelijk aan elkaar.

Reply:
Done